qemu/tests/multi_vms_with_stress.py

```diff
@@ -95,6 +96,14 @@ def wait_for_shutdown_all_vms(vms, sessions):
     run_stress_background()
     is_stress_alive()
 
+    if params.get('set_maxcpus', 'no') == 'yes':
+        num_vms = int(len(params.get('vms').split()))
```

Comment:
fyi: len(params.objects('vms')), the code is OK for me

Reply:
thanks a lot, correct later

Reply:
Corrected